### ISSUE TITLE
Where to find the cifar-100 train/test/solver configure file

### ISSUE THREAD

**Issue by bityigoss** [s9xie/DSN#4]
Would you please upload the cifar-100 net configure file? Thanks!!!

**Comment by limpson** [comment]
Have you tried cifar-100 data set?

**Comment by s9xie** [comment]
Please use the same configuration file for cifar-10 the only thing needed to be changed is using softmax loss instead of hinge loss.

**Comment by feynmanliang** [comment]
Do we need to change `num_output` in cccp6 and the ip supervision layers?

**Comment by s9xie** [comment]
Yes the only thing you need to change is the num of output (10 -> 100). And
you should replace hinge losses with softmax losses.
